src/backend/BSConfig.pm.template

```diff
@@ -171,6 +171,11 @@ our $relsync_pool = {
 #for example "--whole-file" if network faster than disk
 #our $rsync_extra_options = "--whole-file";
 
+# optional source publishing to rsync server
+#our $publishsource_sync = 'rsync://127.0.0.1/sources';
+# optional filter to publish only some areas
+#our $publishsource_filter = [ "openSUSE:.*", "SUSE:.*" ];
```

Comment:
We really should expose those to `/configuration` in the future...

Reply:
it is a perl regexp, this shouldn't be in /configuration.

if you need a check in frontend we can introduce a request route.